### DISCUSSION TITLE
Tumben yg fork ga banyak 🗿

### DISCUSSION THREAD

**Discussion by ERLANRAHMAT** [NzrlAfndi/Charlotte-Linlin#1] [Show and tell]
Ayo om fork kek 🗿

**Comment by FERDIZ-afk** [comment]
> Ayo om fork kek 🗿

Maklum bang, fitur nya masih dikit. biar ngehindari terjadinya nomor kena banned.

**Reply by hdiiofficial** [reply]
Ingfo di invite contributor tak tambah fitur wgwg

**Comment by hdiiofficial** [comment]
Dri pda gw gabut mo ngapain wgwg
